src/global-styles.scss

```diff
@@ -1,14 +1,71 @@
-$example-color: cornflowerblue;
+// gray-700
+// $bg-color: #374151;
 
-.example-class {
-    color: $example-color;
+// zinc-900
+$bg-color: #18181b;
+$text-color: #f8fafc;
+
+// gray-800
+// $card-lv-1-color: #1f2937;
+
+// zinc-700
+$card-lv-1-color: #3f3f46;
+
+
+
+html {
+    background-color: $bg-color;
+    color: $text-color;
+    height: 100%;
+}
+
+body {
+    height: 100%;
+}
+
+.sveltekit-container {
+    height: 100%;
 }
 
 .button {
+    all: unset;
```

Comment:
prefer:
```css
    background-color: transparent;
    color: white;
    ```

Reply:
I've no qualms.

You're probably going to run into a lot of bad CSS from me.  I've always leaned on ui frameworks to do the lifting for me.